components/eventing-controller/pkg/handlers/utils.go

```diff
@@ -261,6 +261,12 @@ func RemoveStatus(sub eventingv1alpha1.Subscription) *eventingv1alpha1.Subscript
 	return desiredSub
 }
 
+func SetStatusAsNotReady(sub eventingv1alpha1.Subscription) *eventingv1alpha1.Subscription {
```

Comment:
"sub" should be a pointer to eventingv1alpha1.Subscription to avoid the "copy" of the actual parameter to the formal parameter during the function call. This "cop" is not a deep copy.

Reply:
I'm just trying to be consistent with the method above it `RemoveStatus`.

Reply:
To me it seemed that by passing by value, it makes it clear (and prevents) that the subscription is mutated.